Source/Core/GeometryPipeline.js

```diff
@@ -2546,6 +2546,10 @@ define([
             case GeometryType.LINES:
                 splitLongitudeLines(instance);
                 break;
+            default:
+                //>>includeStart('debug', pragmas.debug);
```

Comment:
To save a few characters in the released version, I would by the `default:` inside of the pragma when we have them.